client/blocks/post-share/style.scss

```diff
@@ -283,13 +263,7 @@ $section-border: solid 1px darken( $sidebar-bg-color, 5% );
 	margin: 6px 16px 0;
 
 	@include breakpoint( ">480px" ) {
-		margin-right: 24px;
-		margin-left: 24px;
-
-		//TODO Clean-up after PostTypeList is used for /posts/
-		.post-type-list & {
-			margin: 6 24px 0;
-		}
+		margin: 6px 24px 0;
```

Comment:
@michaeldcain can you verify this change? The original looked incorrect to me so I updated it while doing the clean-up.

Reply:
It actually looks like you can remove the entire `.post-share__services-add` selector - it doesn't seem to be used anywhere in the code (it was replaced with `.post-share__services-add-button`, which was even removed in #20912).